webapp/src/pages/ResponseList/components/ResponseListForm.js

```diff
@@ -343,6 +348,14 @@ class ResponseListForm extends Component {
         const columns = this.generateCols();
         const renderReset = this.renderReset();
 
+        function filterText(filter, row) {
+            const id = filter.pivotId || filter.id;
+            return (
+                row[id] !== undefined ?
+                    String(row[id].toLowerCase()).includes(filter.value.toLowerCase()) : true
```

Comment:
Nit: simpler to write:
row[id] == undefined || String(row[id].toLowerCase()).includes(filter.value.toLowerCase())
?